docs/articles/guides/dotnet-new-templates.md

```diff
@@ -5,7 +5,7 @@ name: BenchmarkDotNet templates
 
 # BenchmarkDotNet templates
 
-BenchmarkDotNet provides project templates to setup your benchmarks easily 
+BenchmarkDotNet provides project templates to setup your benchmarks easily.  
```

Comment:
There is only one space, I think there should be a new paragraph here (a second space is needed)